Analysis/HiggsTauTau/src/HTTCategories.cc

```diff
@@ -136,13 +344,22 @@ namespace ic {
       outtree_->Branch("mva_olddm_vtight_2",&lbyVTightIsolationMVArun2DBoldDMwLT_2);
       outtree_->Branch("tau_decay_mode_2",    &tau_decay_mode_2_);
       outtree_->Branch("tau_decay_mode_1",    &tau_decay_mode_1_);
+      
+      outtree_->Branch("jets_totpt",          &jets_totpt_.var_double);
```

Comment:
Do we need these variables in the trees by default, or can they be option controlled behind  do_HLT_Studies_ option (assuming that's what theyre for)?